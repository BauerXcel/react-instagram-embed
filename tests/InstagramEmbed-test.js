import React from 'react'
import renderer from 'react-test-renderer'
import InstagramEmbed from '../src'

// TODO
describe('InstagramEmbed', () => {
  it('Render', () => {
    const component = renderer.create(
      <InstagramEmbed url='https://instagr.am/p/Zw9o4/' />
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
