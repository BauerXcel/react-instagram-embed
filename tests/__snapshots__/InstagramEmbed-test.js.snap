exports[`InstagramEmbed Render 1`] = `
<div
  dangerouslySetInnerHTML={
    Object {
      "__html": null,
    }
  } />
`;
